sale_validity/views/res_config_settings.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2017 Akretion (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+    <record id="res_config_settings_view_form_sale_order_validity" model="ir.ui.view">
+        <field name="name">default.sale.order.validity.sale.settings.form</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="sale.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr='//div[hasclass("o_settings_container")][3]' position="inside">
+                <div class="col-12 col-lg-6 o_setting_box">
+                    <div class="o_setting_left_pane"/>
+                    <div class="o_setting_right_pane">
+                        <label for="default_sale_order_validity_days"/>
+                        <div class="text-muted mt4 mb4">
+                            By default, the validity date of sale orders will be the date of the sale order plus the number of days defined in this field.
+            If the value of this field is 0, the sale orders will not have a validity date by default.
+                        </div>
+                        <div class="text-muted mt4 mb4">
+                            <field name="default_sale_order_validity_days"/>
+                            <label string="Days" style="margin-left: 5px;"/>
```

Comment:
Invalid XML:  ``label`` nodes must have a ``@for``

Reply:
I believe you can put whatever, even leave it empty:
`<label for="" string="Days" style="margin-left: 5px;"/>`